src/Footer/Index.jsx

```diff
@@ -1,45 +1,41 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
-import Grid from 'terra-grid';
-import Text from 'terra-text';
-import List from 'terra-list';
+import BrandFooter from 'terra-brand-footer';
```

Comment:
Are grid, text and list used anywhere else in this project? If not, we can remove them from the dev-dependency list.

Reply:
Grid and text are still used, list isn't in the `package.json` and isn't used anymore.